stock_picking_customer_ref/tests/test_stock_picking_customer_ref.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2015 AvanzOSC - Alfredo de la Fuente
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestStockPickingCustomerRef(common.SavepointCase):
```

Comment:
Could you change this to TransactionCase ?

Reply:
Done